packages/core-web/src/utils/crypto.ts

```diff
@@ -0,0 +1,44 @@
+import * as secp256k1 from 'secp256k1'
```

Comment:
import the functions you're using, instead of `import *`... this prevents tree shaking from removing the unused functions